src/spec/helpers/helpers.js

```diff
@@ -149,3 +149,77 @@ var triggerKeyOnCountryContainerElement = function(key) {
   triggerKey(getCountryContainerElement()[0], "keydown", key);
 };
 /* eslint-enable @typescript-eslint/no-unused-vars */
+
+const moduleCache = new Map();
+
+/**
+ * A very simplistic module loader.
+ * @param {string} path 
+ * @returns {Promise<{exports: any, hasUtils: boolean}>}
+ */
+async function loadCommonJsModule (path) {
+  const url = new URL(path, window.location.href);
+
+  let module = moduleCache.get(url);
+
+  if (!module) {
+    const response = await fetch(url);
+    if (response.status !== 200) {
+      throw new TypeError(`error loading dynamically imported module: ${url}`);
+    }
+
+    const source = await response.text();
+    module = eval?.(`
+      (function() {
+        const module = { exports: {} };
+        ${source}
+        return module;
+      })()
+    `);
+    module.hasUtils = !!module.exports.utils;
+
+    moduleCache.set(moduleCache, module);
+  }
+
+  return module;
+}
+
+/**
+ * Configure a test suite to use a specific build of intl-tel-input.
+ * @param {string} path
+ * 
+ * @example
+ * describe("my suite", function () {
+ *   useIntlTelInputBuild("build/intlTelInputWithUtils.js");
+ *
+ *   it("should do something", () => {
+ *     // In this test, window.intlTelInput will be the version from
+ *     // "build/intlTelInputWithUtils.js"
+ *   })
+ * })
+ */
+function useIntlTelInputBuild(path) {
```

Comment:
I left this in just in case it’s useful for something else while Jasmine tests continue to live. Can remove if it’s in the way.

You can see how this was used in 7a13ace107e7ba1f76833dcdd207e8455d782af4, before all the new tests I added that used this got removed and ported to Jest.

I did not write something similar for Jest because it wasn’t *as* important there — everything is in actual separate modules there so there isn’t global state to worry about in the same way. Having something like this could still simplify Jest tests *slightly*.